src/modeler/AutoRest.Swagger/Model/ServiceDefinition.cs

```diff
@@ -105,6 +105,7 @@ public ServiceDefinition()
         /// <summary>
         /// Key is the object serviceTypeName and the value is swagger definition.
         /// </summary>
+        [Rule(typeof(ArmResourcePropertiesBag))]
```

Comment:
should the rule apply only to definitions or could this properties be defined inline?